docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,49 +1,68 @@
-This is where you can make changes to the element.
-+
-[role="screenshot"]
-image::images/canvas_workpad_edit_style.png[]
+. Open *Canvas*.
 
-. Try changing the color palette. Easy right? Click the Back button in your 
-browser to undo your change.
+. Choose one of the following options:
 
-. Try dragging the chart around the page. Grab the resize handles to make 
-the chart bigger and smaller.
+* To start with a single, blank workpad, click *Create workpad*.
 
-. Now click somewhere off of the element, but not on another element, 
-to deselect it. 
-+
-You will notice the sidebar changing again. You now have 
-access to the page and workpad-level settings. Feel free to change the page 
-background color or rename your workpad.
-
-Speaking of pages, Canvas workpads can have multiple pages.
-
-. Click the arrow to the right of the *Page 1 of 2* text in the toolbar at the bottom
-of the page. You'll see a whole new view of the the eCommerce data.
-. Click the *Page 2 of 2* text to see a live preview of all of the pages in your 
-workpad.
-. Click the (+) button to create a new, blank page.
-. Click the *Add element* button in the top right hand corner of the page.
-. Add an *Area Chart.*
+* To create a workpad from a template, select *Templates*, then select a preconfigured template.
+
+* To import an existing workpad, click and drag a workpad JSON file to the *Import workpad JSON file* field.
+
+For advanced workpad examples, add a {kibana-ref}/add-sample-data.html[sample Kibana data set], then select *Canvas* from the *View Data* drop-down list.
+
+For more workpad inspiration, go to the link:elastic.co[Elastic Blog] where we share Canvas updates and customer stories. 
+
+[float]
+==== Personalize your workpad
+
+To personalize your workpad, use the editor on the right.
+
+. Add a *Name* to your workpad, for example, `My Canvas workpad`.
+
+. In the *Width* and *Height* fields, specify the size of your workpad. 
+
+. Select the layout for your workpad. 
 +
-[role="screenshot"]
-image::images/canvas_workpad_3_page.png[]
+For example, click *720p* for a traditional presentation layout.
+
+. Click the *Background color* picker, then select the background color for your workpad.
+
+image::images/canvas-background-color-picker.gif[]
+
+TIP: To view your workpad in fullscreen mode, click the *Enter fullscreen mode* icon in the upper left corner.
+
+[float]
+=== Share your workpad
+
+When you are ready to share your workpad, create a PDF, or export your workpad to another author.
+
+[float]
+==== Create a PDF of your workpad
+
+To view your workpad outside of Kibana, generate a PDF. 
+
+. Enable reporting in your config/kibana.yml file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Generate PDF*. 
+
+image::images/canvas-generate-pdf.gif[]
+
+[float]
+==== Export your workpad
+
+To import your workpads to another author, create JSON files of your workpads.
```

Comment:
Maybe this should be "To export your workpads to another auther..." the import word confused my brain for a second.

Reply:
Yeah, that tripped me up a little bit too. 

How about: 
"To share your workpads with another author, export your workpads as JSON files that they can import."

Reply:
Since this chunk of content is specifically about exporting, I dropped the end so that it looks like:

`To share your workpads with another author, export your workpads as JSON files.`